packages/client/src/containers/RelatedItems/RelatedItems.Container.js

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import ProductCard from '../../components/ProductCard/ProductCard.component';
+import { apiURL } from '../../apiURL';
+import './RelatedItems.styles.css';
+
+export const RelatedItems = ({ category }) => {
+  const [categoryProducts, setCategoryProducts] = useState([]);
+
+  useEffect(() => {
+    const fetchProductsByCategory = async () => {
+      const response = await fetch(`${apiURL()}/products?category=${category}`);
+      const categoryProductData = await response.json();
+
+      setCategoryProducts(categoryProductData);
+    };
+
+    fetchProductsByCategory();
+  }, [category]);
+
+  return (
+    <div className="related-items">
+      <h1 className="realtedItems-heading">Related Items</h1>
+      <div className="related-products">
```

Comment:
As the Srpint2 is getting a bit stricter, I will ask for some renaming. 🙏🏽 
We need to make up our minds here whether it is related-items or related-products and make the class naming more unified, all the way up to the component itself too.

Reply:
Changed the name to related-items